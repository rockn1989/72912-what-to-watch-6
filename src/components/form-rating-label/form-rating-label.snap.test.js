import React from 'react';
import {render} from '@testing-library/react';
import FormRatingLabel from './form-rating-label';

it(`Should FormRatingLabel render correctly`, () => {
  const data = {
    idx: 1,
    onCheckRatingHandler: () => {},
    formStatus: true
  };

  const {container} = render(<FormRatingLabel {...data}/>);
  expect(container).toMatchSnapshot();
});
