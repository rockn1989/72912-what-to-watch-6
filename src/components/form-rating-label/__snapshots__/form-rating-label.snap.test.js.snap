// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FormRatingLabel render correctly 1`] = `
<div>
  <input
    class="rating__input"
    id="star-2"
    name="rating"
    type="radio"
    value="2"
  />
  <label
    class="rating__label"
    for="star-2"
  >
    Rating 
    2
  </label>
</div>
`;
